Dnn.AdminExperience/Dnn.PersonaBar.Extensions/admin/personaBar/Dnn.SiteImportExport/scripts/bundles/siteimportexport-bundle.js.LICENSE.txt

```diff
@@ -0,0 +1,5 @@
+/*!
+	Copyright (c) 2018 Jed Watson.
+	Licensed under the MIT License (MIT), see
+	http://jedwatson.github.io/classnames
+*/
```

Comment:
Unrelated but probably auto-generated, so we could keep it.

Reply:
I actually deleted this twice and thought it was not in the commit. Surprised to see it again. No clue what step generated it. Build steps are above my pay grade still.

Reply:
Yeah, no big deal, it is handled by CI/Build to generate those license files.